pdc/apps/package/views.py

```diff
@@ -53,10 +53,13 @@ def list(self, *args, **kwargs):
         constraint will not make the package dependencies inconsistent.
 
         For example filtering by `python=2.7.0` would include packages with
-        dependency on `python=2.7.0`, `python>=2.6.0`, `python<3.0.0`, but
-        exclude `python=2.6.0`. Filtering by `python<3.0.0` would include
-        packages with `python>2.7.0`, `python=2.6.0`, `python<3.3.0`, but
-        exclude `python>3.1.0` or `python>3.0.0 && python <3.3.0`.
+        dependency on `python=2.7.0`, `python>=2.6.0`, `python<3.0.0` and
+        `python`, but exclude `python=2.6.0`. Filtering by `python<3.0.0`
```

Comment:
Actually I am a bit confused with what is the output with filter python=2.7.0, why it includes python, why it includes python>=2.6.0

Reply:
Dependency "python>=2.6.0" satisfies filter "python=2.7.0", i.e. it's possible to use python-2.7.0 if dependency specifies that python package version should be greater or equal 2.6.0.

Another example: Dependency "python<3.0.0" satisfies filter "python>=2.7.0", because it's possible to pick some version of python between 2.7.0 (inclusive) up to 3.0.0 (exclusive) -  e.g. python-2.7.0.

NOTE: It's not possible to satisfy dependency "python>2.7.0" with filter "python<2.7.1" (assuming there is no version in-between those two), but the filter still matches the dependency. To handle this properly would require to implement more logic and looks like it's out of scope for PDC.